test/translation/transformation/variableStatementExportAnonymous.ts

```diff
@@ -0,0 +1 @@
+export const [x, , y] = [1, 2, 3];
```

Comment:
We already have a (disabled) test in `destructuring.spec.ts`: https://github.com/TypeScriptToLua/TypeScriptToLua/blob/3373e4a0aab57c475c02bca07d9d7554ae5768e9/test/unit/destructuring.spec.ts#L56